front/app/translations/en.json

```diff
@@ -180,6 +180,9 @@
   "app.components.InitiativeCards.xInitiatives": "{initiativesCount, plural, one {# proposal} other {# proposals}}",
   "app.components.InitiativeForm.bannerUploadLabel": "Add a banner",
   "app.components.InitiativeForm.bannerUploadLabelSubtext": "Make your page visually more appealing. This image will be shown on top of your page.",
+  "app.components.InitiativeForm.cosponsorSectionTitle": "Cosponsors needed for review",
+  "app.components.InitiativeForm.cosponsorSubtextAfterInput": "Your cosponsors will get an email to validate your proposal.",
+  "app.components.InitiativeForm.cosponsorSubtextBeforeInput": "It's required that you have {cosponsorsNumber, plural, one {1 cosponsor} other {# cosponsors}} on your proposals. Add your cosponsors' names below:",
```

Comment:
Typo. "on your proposal**s**" => "on your proposal"

Reply:
Actually the Figma design copy has some more differences and needs to be checked on grammar.